src/test/java/com/zendesk/maxwell/schema/ddl/DDLIntegrationTest.java

```diff
@@ -144,7 +144,32 @@ public void testIntX() throws Exception {
 		};
 
 		testIntegration(sql);
+	}
+
+	@Test
+	public void testYearWithLength() throws Exception {
+		String sql[] = {
+			"create TABLE `test_year` ( id year(4) )"
```

Comment:
Wondering what made them create a YEAR type. What do they do for BC years? negative? Are there no paleontologist+DBAs?